src/dymaptic.GeoBlazor.Core/Scripts/featureLayer.ts

```diff
@@ -235,7 +255,64 @@ export default class FeatureLayerWrapper {
         } else {
             result = await this.layer.applyEdits(jsEdits);
         }
-        
+
         return result;
     }
-}
\ No newline at end of file
+
+    async getFeatureType(graphic: DotNetGraphic): Promise<any> {
+
+        let feature = graphicsRefs[graphic.id as string] as Graphic;
+
+        let result = this.layer.getFeatureType(feature);
+
+        return buildDotNetFeatureType(result);
+    }
+
+    async getField(fieldName: string): Promise<DotNetField | null> {
+
+        let result = await this.layer.getField(fieldName);
+
+        if (result != undefined) {
+            let field = buildDotNetFields(Array(result));
```

Comment:
this `Array(result)` could also be `[ result ]`, right? (I'm just more familiar with that pattern, this is fine).

Reply:
There are some differences, so for consistency, I will use [ ]